### PR TITLE
Add support for reauthorize/edit order

### PR DESCRIPTION
**Pull request by NiklasHogefjord**
* Adds support for edit/reauthorize order.
* Fixes shipping method name displayed in checkout when using delivery module feature.
* New/old checkout API fix.
* Small PHP error fix.

### CODE REVIEW

**Comment by NiklasHogefjord** [comment]
Task linked: [CU-3wnvg82 Feature:  New management API - Reauthorize/edit order](https://app.clickup.com/t/3wnvg82)

File: assets/js/walley-order-meta-box.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+jQuery(function ($) {
+	var walleyAdmin = {
+		init: function () {
+			$('body').on('click', '.sync-btn-walley', walleyAdmin.syncOrderBtn );
+
+		},
+
+		syncOrderBtn:function(e) {
+			e.preventDefault();
```
Comment (MichaelBengtsson) [inline assets/js/walley-order-meta-box.js:9]: If you make the button a type="button" you don't need this. Not required just as a note.

File: templates/walley-checkout-meta-box.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+if ( 'yes' !== $manage_orders ) {
+	return;
+}
+
+if ( in_array( $walley_order_status, array( 'NotActivated', 'PartActivated' ), true ) ) {
+	?>
+	<div class="walley_sync_wrapper">
+		<button class="button-primary sync-btn-walley"><?php esc_html_e( 'Sync order to Walley', 'collector-checkout-for-woocommerce' ); ?></button>
```
Comment (MichaelBengtsson) [inline templates/walley-checkout-meta-box.php:21]: If you make this button a type button you dont need to worry about default behaviour when pressing it inside a form.
`<button type="button" class="button-primary sync-btn-walley"><?php esc_html_e( 'Sync order to Walley', 'collector-checkout-for-woocommerce' ); ?></button>`

**Review by MichaelBengtsson — APPROVED** [review]
LGTM

**Pull request merged**